test/unit-test-helpers/mock_ima.js

```diff
@@ -167,7 +167,7 @@ google =
       {
         ALL_ADS_COMPLETED : "allAdsCompleted",
         COMPLETE : "complete",
-        SKIPPED : "skipped",
+        SKIPPED : "skip",
```

Comment:
Yes, it actually wasn't getting caught originally because we have "skipped", but it's really supposed to be "skip"